app/controllers/application_controller.rb

```diff
@@ -212,6 +214,10 @@ def save_current_host_with_port
   def fetch_community
     @current_community = ApplicationController.find_community(community_identifiers)
 
+    m_community = Maybe(@current_community)
+    logger.community_id = m_community.id
+    logger.community_ident = m_community.ident
```

Comment:
This feels like random global scope mutating. How about a wrapper method to set these to logger instance so it would at least be clear who manages the logger instance:

```ruby
setup_logger(community_id: m_community.id, community_ident: community_ident: m_community.ident)
```